pkgs/applications/networking/cluster/fn-cli/default.nix

```
@@ -0,0 +1,39 @@
+{ pkgs ? import <nixpkgs> {} }:
+
+with pkgs;
+
+buildGoModule rec {
+  pname = "fn";
+  version = "0.5.99";
+
+  name = "${pname}-${version}";
+
+  meta = with stdenv.lib; {
```

```suggestion
  meta = with lib; {
```

please move this to the end.